java/de/yessoft/android/mohammad/ui/MainActivity/fragments/MySpecialORM.java

```diff
@@ -0,0 +1,42 @@
+package de.yessoft.android.mohammad.ui.MainActivity.fragments;
+
+import android.util.Log;
+
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QuerySnapshot;
+
+import java.util.List;
+
+import de.yessoft.android.mohammad.model.NewsItem;
+import de.yessoft.android.mohammad.ui.MainActivity.fragments.NewsFeedRV.NewsFeedAdapter;
+
+class MySpecialORM {
+    private static final String TAG = "ML_MySpecialORM";
+
+    private NewsFeedAdapter RecyclerViewAdapter;
+
+    MySpecialORM(NewsFeedAdapter adapter){
+        RecyclerViewAdapter = adapter;
+    }
+
+    // TODO: Make this Live Observable Pattern -_-
+    void getNewsFeed(){
+        FirebaseFirestore DB = FirebaseFirestore.getInstance();
+
+        // Read Data Once,
+        CollectionReference newsItems = DB.collection("newsfeed");
+
+        newsItems.get().addOnCompleteListener(task -> {
+            QuerySnapshot snapshot = task.getResult();
+            if (task.isSuccessful()) {
```

Comment:
nested if are not allowed

Reply:
The idea is imported from the Official Documentation, now i can move this to a controller.
know what i'm gonna do just that